install.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+
+[ ! -d ~/.tldr ] && printf "tldr seems to have already been previously installed.\nDelete the \'$HOME/.tldr\' directory first before relaunching this script\n" && exit 1 || mkdir -p ~/.tldr/cache
```

Comment:
One of the goals I tried to maintain with the script is that it should "just work" at all times. So the script should clean up after itself and maintain its own caches. This is another benefit of using ~/.config/tldr/, as it is more clearly a directory under the script's control.